src/sentry/static/sentry/less/sentry-hovercard.less

```diff
@@ -71,6 +89,11 @@
     animation-play-state: running;
   }
 
+  &.one-line {
+    width: auto;
+    white-space: nowrap;
```

Comment:
this is in theory fine, thought it could cause a bug if a hovercard had complex formatting of some sort.

Reply:
I tested it with headers and stuff and it seems to work so 🤷‍♀️